### PR TITLE
Parsing of the S message content

### PR DESCRIPTION
**Pull request by marcelrv**
Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: bundles/binding/org.openhab.binding.maxcube/src/main/java/org/openhab/binding/maxcube/internal/message/S_Message.java [hunk 1]
```diff
@@ -9,36 +9,61 @@
...
+		logger.debug("\tCommand Discarded : {}", this.commandDiscarded);
+		logger.debug("\tFreeMemorySlots : {}", this.freeMemorySlots);
 	}
-
 	@Override
 	public MessageType getType() {
 		return MessageType.S;
 	}
-}
+}
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.maxcube/src/main/java/org/openhab/binding/maxcube/internal/message/S_Message.java]: please add newline

Comment (teichsta) [inline bundles/binding/org.openhab.binding.maxcube/src/main/java/org/openhab/binding/maxcube/internal/message/S_Message.java]: and there is still one new line missing ;-)

**Comment by marcelrv** [comment]
@teichsta now I got the right one... I hope :-)

**Comment by teichsta** [comment]
> now I got the right one... I hope :-)

no, unfortunately not ;-) Every file should "end" with a newline in order to make diffs work properly … an there is no new line at the end of S_Message (see red arrow in "files changed" view).

Best, Thomas E.-E.

**Comment by marcelrv** [comment]
Hi Thomas,

ah, did not know that. 
I scanned also the other files and this issue was in quite a few files.

**Comment by teichsta** [comment]
Thanks @marcelrv :-)

**Pull request merged**